persistence/src/test/java/edu/unc/lib/boxc/persist/impl/MultiDigestInputStreamWrapperTest.java

```diff
@@ -0,0 +1,79 @@
+package edu.unc.lib.boxc.persist.impl;
+
+import edu.unc.lib.boxc.persist.api.DigestAlgorithm;
+import edu.unc.lib.boxc.persist.api.exceptions.InvalidChecksumException;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+/**
+ * @author bbpennel
+ */
+public class MultiDigestInputStreamWrapperTest {
+
+    private static String CONTENT = "Something to digest";
+
+    private static String MD5 = "7afbf05666feeebe7fbbf1c9071584e6";
+    private static URI MD5_URI = URI.create("urn:md5:" + MD5);
+
+    private static String SHA1 = "23d51c61a578a8cb00c5eec6b29c12b7da15c8de";
+    private static URI SHA1_URI = URI.create("urn:sha1:" + SHA1);
+
+    private static String SHA512 =
```

Comment:
Lines 32-376 don't appear to be used. Are they needed?

Reply:
Ah yes, thanks, this test was partially copy and pasted from Fedora and I forgot to clean it up